lib/axe/api/a11y_check.rb

```diff
@@ -37,11 +37,18 @@ def call(page)
       private
 
       def audit(page)
-        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", @context.to_json, @options.to_json
+        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", *js_args
+      end
+
+      def js_args
+        [ @context, @options ]
+          .reject{ |arg| arg.empty? }
```

Comment:
FWIW It's documented as an optional argument in axe-core now: https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axerun